app/controllers/api/v1/road_trip_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::V1::RoadTripController < ApplicationController
+  def create
+    if User.find_by(api_key: params[:api_key])
```

Comment:
wanted to implement a current_user type of method here, but couldnt figure out a way to get an application controller method to work in these controllers.